templates/sdk-github-issue-template.yaml

```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
```

Comment:
I'm not against using the "triage" label but if we're gonna go for that we ought to add it to https://github.com/ably/engineering/blob/main/sdk/github.md#labels and document some process about how we use it - i guess that would just be "remove the label once the bug is prioritised"